statuspage/src/components/CanaryModal.vue

```diff
@@ -0,0 +1,60 @@
+<!-- This component encapsulates a modal that is displayed -->
+<!-- when clicking on canary                               -->
+<template>
+  <b-modal :id="'modal-canary'+name+namespace" ok-only>
+    <template #modal-title>
```

Comment:
We'll need a way of maintaining accessibility if we're going to use this approach. We need to ensure that the `aria-labelledby` attribute points at an appropriate node: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-labelledby_attribute

See bootstrap notes: https://bootstrap-vue.org/docs/components/modal#accessibility

Reply:
Can we add accessibility handling into the style guide?  It might be something we explicitly defer until we have a hard requirement and/or VC funding

Reply:
@moshloop It's not easy to bolt on after the fact so would be ideal if it could be part of habitual best practice. 

What do you mean add it into the style guide in this case?

Either way, I did some research last night, and in this case if we make the `alt` tag on the image `alt=""` then screen reader should skip it when reading, so instead of reading `Heading Level 5 undefined image <title>` it would just read `Heading Level 5 <title>`, which was the behaviour from the unmodified component.